frontend/src/components/ConfirmationDialog.scss

```
@@ -2,33 +2,55 @@
   position: fixed;
   top: 0;
   left: 0;
-  width: 100%;
-  height: 100%;
+  bottom: 0;
+  right: 0;
   background-color: rgba(0, 0, 0, 0.5);
   display: flex;
   justify-content: center;
   align-items: center;
   z-index: 100;
 }
 
-.confirmation-dialog-container {
-  background-color: white;
-  padding: var(--layout-padding-large);
+.confirmation-dialog-outer {
+  display: grid;
+  grid-template-columns: 1% 99%;
```

You can actually accomplish that yellow edge effect just with something like
```css
  border-left: 1rem solid var(--support-border-color-warning);
```

This approach works too, but the percentages will make its size change on different screens, so let's hardcode it to a constant value (looks like 5px in Figma)

```suggestion
  grid-template-columns: 5px 1fr;
```